ReactWindows/ReactNative.Shared/ReactInstanceManager.cs

```diff
@@ -277,9 +278,6 @@ public void OnSuspend()
             }
 
             _lifecycleStateMachine.OnSuspend();
-
-            ReactChoreographer.Dispose();
-            DispatcherHelpers.Reset();
         }
```

Comment:
I guess my concern is that we are using the same Dispatcher across suspend/resume cycles. If it's actually the case that the dispatcher does not change over suspend/resume cycles, then this is not a problem. If RNW is inadvertently keeping the Dispatcher alive when it does not naturally happen in UWP, then we do have a problem.

Reply:
The Dispatcher / ,main view/ various windows / etc., nothing changes. A suspended up is just a frozen one. The "Application" object is the same when resuming, everything is the same.

---
In reply to: [170011967](https://github.com/Microsoft/react-native-windows/pull/1653#discussion_r170011967) [](ancestors = 170011967)